src/ugen_xxx.cpp

```diff
@@ -970,258 +972,329 @@ DLL_QUERY lisa_query( Chuck_DL_Query * QUERY )
     func->add_arg( "int", "voice" );
 	if( !type_engine_import_mfun( env, func ) ) goto error;
     
 	// set/get voicePan [value between 0 and numchans-1, to place voice]
     func = make_new_mfun( "float", "voicePan", LiSaMulti_ctrl_voicepan );
+    func->doc = "For particular voice (arg 1), set panning value [0.0, number of channels - 1.0].";
     func->add_arg( "int", "voice" );
     func->add_arg( "float", "val" );
     if( !type_engine_import_mfun( env, func ) ) goto error;
     func = make_new_mfun( "float", "voicePan", LiSaMulti_cget_voicepan);
+    func->doc = "For particular voice (arg 1), get panning value.";
     func->add_arg( "int", "voice" );
 	if( !type_engine_import_mfun( env, func ) ) goto error;
 	
 	// set/get voicePan [value between 0 and numchans-1, to place voice]
     func = make_new_mfun( "float", "pan", LiSaMulti_ctrl_voicepan );
+    func->doc = "For particular voice (arg 1), set panning value [0.0, number of channels - 1.0].";
     func->add_arg( "int", "voice" );
     func->add_arg( "float", "val" );
     if( !type_engine_import_mfun( env, func ) ) goto error;
     func = make_new_mfun( "float", "pan", LiSaMulti_cget_voicepan);
+    func->doc = "For particular voice (arg 1), get panning value.";
     func->add_arg( "int", "voice" );
 	if( !type_engine_import_mfun( env, func ) ) goto error;
 	
 	func = make_new_mfun( "float", "pan", LiSaMulti_ctrl_voicepan0 );
+    func->doc = "Set panning value [0.0, number of channels - 1.0].";
     func->add_arg( "float", "val" );
     if( !type_engine_import_mfun( env, func ) ) goto error;
     func = make_new_mfun( "float", "pan", LiSaMulti_cget_voicepan0);
+    func->doc = "Get panning value.";
 	if( !type_engine_import_mfun( env, func ) ) goto error;
 	
     // set record feedback coefficient
     func = make_new_mfun( "float", "feedback", LiSaMulti_ctrl_coeff );
+    func->doc = "Set feedback amount when overdubbing (loop recording; how much to retain).";
     func->add_arg( "float", "val" );
     if( !type_engine_import_mfun( env, func ) ) goto error;
     func = make_new_mfun( "float", "feedback", LiSaMulti_cget_coeff);
+    func->doc = "Get feedback amount when overdubbing (loop recording; how much to retain).";
     if( !type_engine_import_mfun( env, func ) ) goto error;
     
     // clear buffer
     func = make_new_mfun( "void", "clear", LiSaMulti_ctrl_clear );
+    func->doc = "Clear recording buffer.";
     if( !type_engine_import_mfun( env, func ) ) goto error;
     
     // get free voice
     func = make_new_mfun( "int", "getVoice", LiSaMulti_cget_voice );
+    func->doc = "Return an available voice (one that is not currently playing). Return -1 if no voice is available.";
     if( !type_engine_import_mfun( env, func ) ) goto error;
     
     // set maxvoices
     func = make_new_mfun( "int", "maxVoices", LiSaMulti_ctrl_maxvoices );
+    func->doc = "Set the maximum number of voices allowable; 10 by default (200 is the current hardwired internal limit).";
```

Comment:
This is now 256 https://github.com/ccrma/chuck/blob/949b7e8472a746820dbef71d93a6d0b144d59287/src/core/ugen_xxx.cpp#L3989